src/go/monitor/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+func main() {
+	http.Handle("/metrics", promhttp.Handler())
```

Comment:
Do we want to add a simple visit counter middleware for the entire app, to show how to expose a metric?

Reply:
I will add a metric to count the total of visit.
To keep in simple, for example purpose, I propose to keep a single `main.go`file.